data/quarg/quarg news.txt

```diff
@@ -469,6 +469,7 @@ news "ringworld quarg"
 		attributes "quarg"
 		attributes "station"
 		not planet "Lagrange"
+		not planet "Ruelogakk"
```

Comment:
```suggestion
		attributes "ringworld"
```

Reply:
Can this be on the same line as the quarg attribute?

Reply:
I think putting them on the same line makes it an `or` rather than `and`.
For example, the destination for `Escort (Tiny, Northern, Dangerous Origin)` has the condition `attributes "deep" "paradise" "near earth" "core"`, and hence can be any planet that has at least one of those attributes.

In this case, though, we specifically want this news to only appear on Quarg ringworlds, not just anywhere that's either Quarg or a ringworld, so we need the two to be on separate lines.